assets/js/components/channels/forms/AzureHubForm.jsx

```diff
@@ -58,14 +58,23 @@ class AzureHubForm extends Component {
       } else {
         const credentials = {};
         e.target.value.split(";").forEach((x) => {
-          const [key, value] = x.split("=");
+          const tuple = x.split("=");
```

Comment:
I was going to fix by just doing `.split(/=(.*)/s);` instead which will only look at the first occurrence of `=`. Might make it a little more straightforward to read this for posterity. Thoughts?

Reply:
tested:

```
➜  console git:(dev) ✗ node
Welcome to Node.js v14.15.5.
Type ".help" for more information.
> let credentials = {};
undefined
> let someString = "HostName=myname;SharedAccessKeyName=word;SharedAccessKey=wow=2482";
undefined
> let someFunction = (input) => { input.split(";").forEach((x) => {
...           const [key, value] = x.split(/=(.*)/s);
...           if (
.....             ["HostName", "SharedAccessKeyName", "SharedAccessKey"].indexOf(
.......               key
.......             ) > -1 &&
.....             valueOf
.....           ) {
.....             credentials[key] = value;
.....           }
...         });};
undefined
> someFunction(someString);
undefined
> credentials
{
  HostName: 'myname',
  SharedAccessKeyName: 'word',
  SharedAccessKey: 'wow=2482'
}
```

Reply:
Great idea, il change it out